gslib/boto_translation.py

```diff
@@ -405,8 +405,15 @@ def ListObjects(self, bucket_name, prefix=None, delimiter=None,
           key_to_convert = key
 
           # Listed keys are populated with these fields during bucket listing.
-          key_http_fields = set(['bucket', 'etag', 'name', 'updated',
-                                 'generation', 'metageneration', 'size'])
+          key_http_fields = {
```

Comment:
I'd argue that this decreases readability.  When I see curly braces, I immediately jump to thinking it's a `dict`.   Also, there's no syntax to initialize an empty `set` like this, (`{}` would create a `dict`), and I'd rather be consistent with how we initialize `set`s across the codebase.

Reply:
There are many places where sets are defined only with curley brackets. Should they all be changed to set([])?

Reply:
Discussed offline - in order to get caught up with local branches, I'm willing to pull the "set display" syntax in, but I'd like to change it back to use the explicit `set([...])` syntax once we've got this branch caught up.

Reply:
As per chat, these changes will be re-addressed at a later time.